src/pages/PeoplePage/PeopleTable.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PersonComponent from '../../components/PersonComponent';
```

Comment:
thank you. So I tested the application and all the tests pass. I think the extension is unnecessary. Because in all cases I write imports to components without extensions.